graphql/multiplication-connect/createUserData.ts

```diff
@@ -0,0 +1,9 @@
+import { gql } from "@apollo/client";
+
+export const CREATE_USER_MC_DATA = gql`
+  mutation createUser($id: String = "wHZGkjB7CFf0cMrcGgI1pIHfke02") {
```

Comment:
probably shouldn't have your user id hardcoded here. I know I've done that in the past, but it isn't secure to have this here.